packages/bpk-component-calendar/src/BpkCalendarDate.js

```diff
@@ -155,6 +155,8 @@ BpkCalendarDate.propTypes = {
   preventKeyboardFocus: PropTypes.bool,
 };
 
+export const BpkCalendarDatePropTypes = BpkCalendarDate.propTypes;
```

Comment:
This will not work with an optimizer that removes these in production. Instead of doing it this way declare `PropTypes` as constant, export that and assign it to `BpkCalednarDate.propTypes`